new-integration-tests/src/test/java/oracle/weblogic/kubernetes/assertions/impl/Domain.java

```diff
@@ -64,9 +67,9 @@ public static boolean doesCrdExist() throws ApiException {
   /**
    * Checks if weblogic.oracle CRD domain object exists.
    *
-   * @param domainUid domain UID of the domain object
+   * @param domainUid     domain UID of the domain object
    * @param domainVersion version value for Kind Domain
-   * @param namespace in which the domain object exists
+   * @param namespace     in which the domain object exists
```

Comment:
The sentence should be read as follows ...
@param namespace  namespace  in which the domain object exists

The first word is the exact name of the parameter and the rest of the sentence is to describe the parameter